c/tskit/trees.c

```diff
@@ -407,6 +407,10 @@ tsk_treeseq_init(
     tsk_memset(self, 0, sizeof(*self));
     if (options & TSK_TAKE_OWNERSHIP) {
         self->tables = tables;
+        if (tables->edges.options & TSK_NO_EDGE_METADATA) {
```

Comment:
Shouldn't the flag be TSK_TABLE_NO_METADATA though? I thought that was the point of that flag - any table that has no metadata would use it. the TSK_TC_NO_EDGE_METADATA was a way to send that flag to edge_table_init.

Maybe we should wait until #2224 is in before tackling this.

Reply:
Ah, they have the same value as I haven't rebased, so if I had rebased my mistake would have been revealed. 

I've rebased and fixed this.